workers/unity/Packages/manifest.json

```diff
@@ -16,13 +17,15 @@
     "io.improbable.gdk.transformsynchronization": "0.3.10",
     "com.unity.modules.ai": "1.0.0",
     "com.unity.modules.animation": "1.0.0",
+    "com.unity.modules.assetbundle": "1.0.0",
```

Comment:
what is this?

Reply:
Part of the engine that's been moved into packages I think. If you don't have this, you get errors saying a type has been redirected and to install this.